app-modules/ai/src/Filament/Resources/AiAssistantResource/Pages/ListAiAssistants.php

```diff
@@ -77,7 +77,8 @@ public function table(Table $table): Table
                 EditAction::make(),
             ])
             ->emptyStateHeading('No AI Assistants')
-            ->emptyStateDescription('Add a new custom AI Assistant by clicking the "Create AI Assistant" button above.');
+            ->emptyStateDescription('Add a new custom AI Assistant by clicking the "Create AI Assistant" button above.')
+            ->modifyQueryUsing(fn (Builder $query) => $query->whereisDefault(false));
```

Comment:
Let's never use the `whereX` magic methods.

Let's either add a tappable scope (https://juststeveking.uk/articles/tappable-query-scopes-in-laravel/) or just write out the where like normal (`->where('is_default', false)`